lib/cog/pipeline/util.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Cog.Pipeline.Util do
+
+  alias Cog.Pipeline.Destination
+
+  def here_destination(request) do
```

Comment:
We still need it so we send correctly formatted output. `:chat`, `:status`, and `:status_only` outputs are all different to varying degrees.

Code was originally moved to `Util` as part of a previous refactoring. I'll see if it can be moved into `Destination` directly.